src/elements/base.js

```diff
@@ -8,6 +8,7 @@ export const defaults = {
   borderWidth: 1,
   outlierRadius: 2,
   outlierColor: Chart.defaults.global.elements.rectangle.backgroundColor,
+  medianColor: Chart.defaults.global.elements.rectangle.borderColor,
```

Comment:
can you implement that the default `medianColor` is the specified `borderColor`?. Such that it is backward compatible. Like when users configured the `borderColor` it would change the appearance right now, since the `medianColor` is not automatically changed.

Reply:
`medianColor` now defaults to `null` which just assumes the current `borderColor` unless overridden.